Source/ProceduralFairings/FairingBase.cs

```diff
@@ -139,14 +139,17 @@ public enum BaseMode { Payload, Adapter, Plate, Other }
         float lastHeight = -1000;
         float lastExtraHt = -1000;
 
+        /// for non-inline fairings, the "real" maximum width of the fairing, the bulge in the middle
```

Comment:
Oh, sorry. I meant this:
```
/// <summary>
/// This is a comment.
/// </summary>
```
Usually VS will insert the stub automatically once you write 3 forward slashes.